client/packages/system/src/Patient/CreatePatientModal/PatientFormTab.tsx

```diff
@@ -12,6 +12,10 @@ import { ObjUtils } from '@common/utils';
 import defaultPatientSchema from '../DefaultPatientSchema.json';
 import defaultPatientUISchema from '../DefaultPatientUISchema.json';
 import { BasicSpinner } from '@openmsupply-client/common';
+import {
+  IdGenerator,
+  idGeneratorTester,
+} from 'packages/programs/src/JsonForms/components';
 
```

Comment:
```suggestion
import { IdGenerator, idGeneratorTester } from '@openmsupply-client/programs';
```

Same here. But you also have to export these symbols in program package index.ts to make it work...